lib/rsteamshot/screenshot.rb

```diff
@@ -92,6 +92,23 @@ def inspect
       self.class.name + '<' + JSON.generate(to_h) + '>'
     end
 
+    # Public: file_size parsed into bytes
+    #
+    # Returns a Integer
+    def file_size_in_bytes
+      unit_multiplier = 1000
+      size, unit = self.file_size.split(' ')
+      bytes = begin
```

Comment:
I don't think the `begin`/`end` is necessary. You can just have `bytes = case unit.downcase` here.